modules/340-monitoring-kubernetes/templates/kube-state-metrics/additional-scrape-config.yaml

```diff
@@ -16,6 +16,23 @@ spec:
     insecureSkipVerify: true
   staticConfigs:
     - targets: ['kube-state-metrics.d8-monitoring.svc.{{ .Values.global.discovery.clusterDomain }}.:8080']
+  metricRelabelings:
+  - action: replace
+    regex: kube_verticalpodautoscaler_labels_info
+    replacement: kube_verticalpodautoscaler_labels
+    sourceLabels:
+    - __name__
+    targetLabel: __name__
+  - sourceLabels: [__name__]
+    regex: kube_verticalpodautoscaler_labels
+    targetLabel: name
+    replacement: ''
```

Comment:
I would suggest using the same pattern for readability:
* action is not required here
* keys appear in a different order
* `sourceLabels` written using different yaml-syntax